Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/JsonConverters/SerializableStroke.cs

```diff
@@ -15,7 +17,11 @@ internal class SerializableStroke
         [JsonIgnore]
         public List<InkPoint> FinalPointList { get; set; }
 
-        public InkDrawingAttributes DrawingAttributes { get; set; }
+        [JsonIgnore]
+        public InkDrawingAttributes DrawingAttributesIgnored { get; set; }
+
+        // This class is created to avoid breaking Changes
+        public CustomInkDrawingAttribute DrawingAttributes { get; set; }
```

Comment:
Should it be marked obsolete?, why the two different DrawingAttributes?

Reply:
That's an internal class. the problem I had is InkDrawingAttributes can't be instantiated using Newtonsoft library in release mode, it was working well in debug and there is a public parameter less constructor but weirdly Newtonsoft can't deserialize it. so I had to created a new DTO class with the same properties and same name to not breaking changes and map it to the DrawingAttributesIgnored which is actually used to construct the InkStroke.
Maybe DrawingAttributesIgnored should be renamed to something else ?

Reply:
This doesn't make sense... Why would it not serialize the class?!?

Reply:
What's the error we're getting from Newtonsoft?

Reply:
@skendrot I know but it work well in debug mode and I get unable to create instance in release mode !!

@michael-hawker I will have a look about JsonConverter.

Reply:
I've a problem with almost every attribute, for example, the Size  when I try to use JSON converter I receive this message Could not determine JSON object type for type Windows.Foundation.Size and all that issues are in release mode, debug is working more than fine.

I see the current way is the easiest way and has fewer impediments.

Reply:
@nmetulev  the updates should be from @michael-hawker 's side 😄

Reply:
The current PR is working as expected

Reply:
This seems to work, and it looks like .net native has some issues otherwise. I'm ok with the compromise.